vue-frontend/src/pages/jobs/apply/[id].vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <div>
+  <div @click="referenceList($event)">
```

Comment:
Why do we need to add it here?

Reply:
If drop-down list is open and click on page that time disable that list

Reply:
I am not getting you.
Initially its false, then it will automatically disable, Why do we need to disable it as a click. Or if it is the use case, then this is not proper solution.